app/src/main/java/fr/free/nrw/commons/notification/NotificationActivity.java

```diff
@@ -147,8 +148,11 @@ private void initListView() {
             refresh(false);
         }
         adapter = new NotificatinAdapter(item -> {
-            Timber.d("Notification clicked %s", item.getLink());
-            handleUrl(item.getLink());
+            if (item.getNotificationType() == NotificationType.EMAIL){
+                ViewUtil.showLongSnackbar(binding.container, "Check your mail box");
```

Comment:
Better write "email inbox" fully to make it 100% explicit.
Also, can you internationalize the string?

Reply:
All good, but Can I have some clarifications for "internationalize the string".

Reply:
The snackbar should be shown in French to users who use the app in French, for instance. :-)
https://developer.android.com/guide/topics/resources/localization

Example: https://github.com/commons-app/apps-android-commons/blob/main/app/src/main/java/fr/free/nrw/commons/notification/NotificationActivity.java#L104